10_fibonacci/README.md

```diff
@@ -2,9 +2,9 @@
 
 Create a function that returns a specific member of the Fibonacci sequence:
 
-> A series of numbers in which each number ( Fibonacci number ) is the sum of the two preceding numbers. The simplest is the series 1, 1, 2, 3, 5, 8, etc.
+> A series of numbers in which each number ( Fibonacci number ) is the sum of the two preceding numbers. The simplest is the series 0, 1, 1, 2, 3, 5, 8, etc.
 
 ```javascript
-fibonacci(4) // returns the 4th member of the series: 3  (1, 1, 2, 3)
-fibonacci(6) // returns 8
+fibonacci(4); // returns the 4th member of the series: 3  (1, 1, 2, 3)
+fibonacci(6); // returns 8
```

Comment:
Shouldn't these be 2 and 5 ?
@thatblindgeye

Reply:
@ManonLef Yes. You are right. I'm sorry I should have seen this.
Doesn't this mean that all the tests have to be changed as well?

Reply:
@ManonLef  In terms of the solution/output we're asking for then yeah. 

Rather than updating the solution and test files, we _could_  just revert the README update and just add a note that mentions the Fibonacci sequence can also start at 0 and 1 (maybe even link to the Wikipedia article about it).

Reply:
@thatblindgeye I don't have a preference really, it would be the least work for now to do it that way.

Reply:
@RushilJalal would you be interested in making the above update?

Reply:
@thatblindgeye Yes I would be glad to make the changes.
What updates am I supposed to make?

Reply:
@RushilJalal I'm pretty confident it would be this :) :
> we could just revert the README update and just add a note that mentions the Fibonacci sequence can also start at 0 and 1 (maybe even link to the Wikipedia article about it).

Reply:
@ManonLef Hey. I have made the required changes but I'm not sure what to do next.
Should I create a new PR or commit in this PR?
I tried the latter but couldn't figure out how to do it. Please advise. 
This is the `README.md`  after the changes I made:
```
# Exercise 10 - Fibonacci

Create a function that returns a specific member of the Fibonacci sequence:

> A series of numbers in which each number ( Fibonacci number ) is the sum of the two preceding numbers.
> In this exercise, the Fibonacci sequence used is 1, 1, 2, 3, 5, 8...
> To learn more about Fibonacci sequences, go to: https://en.wikipedia.org/wiki/Fibonacci_sequence

```javascript
fibonacci(4); // returns the 4th member of the series: 3  (1, 1, 2, 3)
fibonacci(6); // returns 8
```
P.S ignore the _missing triple backticks_ in the last line.
Is there anything else I should change? Lemme know. Thanks.

Reply:
You'll want to open a new PR for this